app/views/shared/check_answers/_restrictions.html.erb

```diff
@@ -1,24 +1,25 @@
-<h2 class="govuk-heading-m"><%= t(".#{journey_type}.heading") %></h2>
-<dl id="restrictions-on-clients-assets-questions" class="govuk-summary-list govuk-!-margin-bottom-0">
-  <%= check_answer_link(
-        name: :restrictions,
-        url: check_answer_url_for(journey_type, :restrictions, @legal_aid_application),
-        question: t(".#{journey_type}.question"),
-        answer: yes_no(@legal_aid_application.has_restrictions),
-        no_border: @legal_aid_application.has_restrictions?,
-        read_only:,
-      ) %>
-
-  <% if @legal_aid_application.has_restrictions %>
-      <%= check_answer_link(
-            name: :restrictions_details,
-            url: check_answer_url_for(journey_type, :restrictions, @legal_aid_application),
-            question: "Details of restrictions",
-            answer: @legal_aid_application.restrictions_details,
-            no_border: @legal_aid_application.has_restrictions?,
-            read_only:,
+<h2 class="govuk-heading-m"><%= t(".#{journey_type}.heading#{individual}") %></h2>
+<%= govuk_summary_list(classes: "govuk-!-margin-bottom-9", html_attributes: { id: "restrictions-on-clients-assets-questions" }) do |summary_list| %>
+  <%= summary_list.with_row(html_attributes: { id: "app-check-your-answers__restrictions" }) do |row| %>
+    <%= row.with_key(text: t(".#{journey_type}.question"), classes: "govuk-!-width-one-half") %>
+    <%= row.with_value { yes_no(@legal_aid_application.has_restrictions) } %>
+    <% unless read_only %>
+      <%= row.with_action(
+            text: t("generic.change"),
+            href: check_answer_url_for(journey_type, :restrictions, @legal_aid_application),
+            visually_hidden_text: t(".#{journey_type}.question"),
           ) %>
+    <% end %>
+  <% end %>
+  <%= summary_list.with_row(html_attributes: { id: "app-check-your-answers__restrictions_details" }) do |row| %>
```

Comment:
This doesn't seem to have the guard clause from line 12 of the old version.
It looks like it will _always_ show the restrictions detail block where it used to be hidden if the first answer was no?